api/api.proto

```diff
@@ -710,8 +710,10 @@ message Volume {
   google.protobuf.Timestamp attach_time = 24;
   // DetachTime time this device was detached.
   google.protobuf.Timestamp detach_time= 25;
+  // LastFsckScanFix is the time when an integrity check fixed errors in filesystem
+  google.protobuf.Timestamp last_fsck_scan_fix = 26;
```

Comment:
you may want to move this to 27,  and retain fpConfig at 26 to retain backward compatibility with clients

Reply:
moved the new field to 27